clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -79,6 +87,22 @@
 
     private final Time time;
 
+    private final Map<Integer, Collection<ApiVersionsResponse.ApiVersion>> nodeApiVersions = new HashMap<>();
+
+    public NetworkClient(Collection<ApiVersionsResponse.ApiVersion> apis,
```

Comment:
It is a bit hard to tell from the diff, but did you add a new argument as the first in the list? Is there a reason? We usually add new arguments at the end.